tmpl/parse.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 the lostromos Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tmpl
+
+import (
+	"io"
+	"text/template"
+)
+
+// Parse will take a CustomResource, template directory and an io.Writer and
+// print the resulting templates to the io.Writer
+func Parse(cr *CustomResource, dir string, r io.Writer) error {
```

Comment:
should 'r' be renamed to 'w'? r would suggest reader to me.

Reply:
Good catch....I think that is a leftover from how it was before.